acctest/pluginacc.go

```diff
@@ -25,6 +25,9 @@ const TestEnvVar = "PACKER_ACC"
 type PluginTestCase struct {
 	// Init, if true `packer init` will be executed prior to `packer build`.
 	Init bool
+	// BuildExtraArgs is the list of arguments to be passed as extra to the packer build
+	// command, before `--machine-readable`
```

Comment:
Good call. I do wonder if we should just state that by default `-machine-readable` is auto appended to this list to prevent someone from adding it twice.